ov/ov.go

```diff
@@ -44,6 +46,33 @@ func (c *OVClient) NewOVClient(user string, password string, domain string, endp
 			IfMatch:    ifmatch,
 		},
 	}
+	apiver, err := c.GetAPIVersion()
+	//If no api version is provided use the current version to create client
+	if apiversion == 0 {
```

Comment:
I am not sure about the case where we get apiversion as 0

Reply:
If no apiverison is given in config file 0 stored in the config struct